Description: Declaring math lib and dynamic loaded lib to LIBS to fix FTBFS.
Author: Leo Iannacone <l3on@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/913459
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=655216
Forwarded: no
---
 configure.ac |    2 ++
 1 file changed, 2 insertions(+)

--- jack-tools.orig/configure.ac
+++ jack-tools/configure.ac
@@ -50,6 +50,8 @@ AC_CHECK_HEADERS(sys/inotify.h, AC_DEFIN
 
 AC_SUBST(AM_CFLAGS)
 AM_CFLAGS="-D_POSIX_C_SOURCE=200112 -std=c99 -O3 -funroll-loops -Wall"
+AC_SUBST(LIBS)
+LIBS="-lm -ldl"
 
 # Write Makefile.
 
